src/ducks/sharing/WhoHasAccess.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import Recipient from '../../components/Recipient'
+import { getRecipients } from '.'
+
+const WhoHasAccess = ({ recipients }) => (
+  <div>
+    {recipients.map(({email, url, status}) => (
+      <Recipient name={email} url={url} status={status} />
+    ))}
+  </div>
+)
+
+class StatefulWhoHasAccess extends React.Component {
+  state = {
+    recipients: []
+  }
+
+  componentDidMount () {
+    const { _id, _type } = this.props.document
+    const { t } = this.context
+    getRecipients(_id, _type)
+    .then(recipients =>
+      recipients.map(recipient => {
+        const status = recipient.status === 'accepted'
+         ? t(`Share.status.${recipient.status}.${recipient.type}`)
+         : t(`Share.status.${recipient.status}`)
+        return {
+          ...recipient,
+          status
+        }
+      })
+    )
+    .then(recipients => {
+      this.setState(state => ({ ...state, recipients }))
+    })
+    .catch(console.error.bind(console))
```

Comment:
In this case I don't think there's much to display, so yeah, logging it so it shows up in sentry is probably a good idea. We're just not very consistent with it :-)